src/main/java/calculator/StringAddCalculator.java

```diff
@@ -31,9 +35,16 @@ public static String[] split(String text) {
     public static int sum(String[] numbers) {
         int result = 0;
         for (String number : numbers) {
-            ValidationChecker.validationCheck(number);
+            validationCheck(number);
+
             result += Integer.parseInt(number);
```

Comment:
number를 감싸는 Number라는 객체를 만들어보는건 어떨까싶네요 👍 그럼 다음과 같이도 할 수 있어요. 

```suggestion
  Number result = new Number();
  for (String number : numbers) {          
      result.plus(new Number(number));
  }
  
  return result;
```
올바른 숫자인지, 양수인지도 계산기가 판단할 필요가 없죠 그에 대한 책임은 더 적절한 Number라는 객체에게 넘어갔을테니까요